scripts/feature_tests/setup_env.sh

```diff
@@ -11,3 +11,8 @@ if [[ "${1}" == "ug" ]]; then
 fi
 pushd "${M3PATH}" || exit
 make
+
+if [[ "${1}" != "ug" ]]; then
+  export ACTION="inspection"
+  "${M3PATH}"/scripts/run.sh
+fi
```

Comment:
Setup_env is supposed to setup the env only, Not sure if calling a test script is a good idea here ....

Reply:
Good point! Updated with by adding a new entry in the Makefile. Thanks for suggestions.